easybuild/easyconfigs/m/mustache-hic/mustache-hic-1.3.3-foss-2023b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonPackage'
+
+name = 'mustache-hic'
```

Comment:
@pavelToman Why not just use `Mustache` here as software name?

`mustache-hic` is just what's it's called on PyPI, but that's a workaround for a naming clash with https://pypi.org/project/mustache, which seems like a dead project...